app/src/main/java/com/github/dedis/student20_pop/utility/network/PoPClientEndpoint.java

```diff
@@ -60,4 +65,37 @@ public void onMessage(String message, Session session) {
     public void onClose(Session session, CloseReason reason) {
         listeners.remove(session);
     }
+
+    static class SimpleDecoder implements Decoder.Text<String> {
```

Comment:
Why do you have twice the SimpleDecoder class ?

Reply:
The class was not needed so I removed it.